src/scripts/main.js

```diff
@@ -1 +1,24 @@
 'use strict';
+
+const nav = document.querySelector('.list-item');
+const imgLarge = document.querySelector('.gallery__large-img');
+const a = document.querySelector('a');
+const img = document.querySelector('.gallery__thumb');
+
+a.addEventListener('click', () => {
+  event.preventDefault();
+});
+
+img.addEventListener('click', () => {
+  event.preventDefault();
+});
+
+nav.addEventListener('click', () => {
+  const link = event.target.closest('.list-item__link', '.gallery__img');
+
+  if (!link) {
+    return;
+  }
+
+  imgLarge.src = link.src;
```

Comment:
The `link.src` might not be the correct way to get the image source. If `link` is an anchor element, you might want to use `link.href` or if it's an image, ensure it's correctly selected.